lnst/RecipeCommon/Perf/Measurements/Results/XDPBenchMeasurementResults.py

```diff
@@ -0,0 +1,30 @@
+from lnst.RecipeCommon.Perf.Results import ParallelPerfResult
+from lnst.RecipeCommon.Perf.Measurements.Results.FlowMeasurementResults import FlowMeasurementResults
+from lnst.RecipeCommon.Perf.Measurements.MeasurementError import MeasurementError
+
+
+class XDPBenchMeasurementResults(FlowMeasurementResults):
```

Comment:
> Could we just reuse the normal `FlowMeasurementResults` class and simply report 0 for the `cpu_stats`?

We cannot, methods that handle warm-up/down slicing and aggregating expect `cpu_stats` to be an instance of `PerfInterval` with non-zero duration (because of division and slicing).
Sure, we can put there `PerfInterval` with duration set to `1`, value `0` and timestamps to neutral values in regard to alignment/warm-up/down slicing, but I don't like the idea of having this kind of "workarounds" in production code.

> The reason being is that i think this doesn't actually contribute any new code, just... masks out the unused cpu stats.

For now, it just masks out cpu stats but in the future, there might be also another metrics e.g. for xdp forward recipe, and we'll end up with separate results anyway.

Reply:
> well i'd say duration should reflect the actual duration in this case... and technically that's still correct

I don't agree, it's not correct. xdp rules are utilizing CPU, but we cannot easily measure how. Setting `receiver.cpu_stats` to `0` would imply that xdp rules are not basically do anything, which you can't conclude from results with `receiver.cpu_stats` being undefined/`None`. 

> xdp flow measurement could add support for the cpu utilization

Discussed offline, since XDP is running as part of kernel instead of a separate process, there is no simple way to measure utilization, that wouldn't significantly affect performance.

We can gather running time statistics via `bpftool prog` with `kernel.bpf_stats_enabled` enabled however, from my experiments it seems that the amount of dropped packets dropped by 30% along with CPU usage being 20% higher.